benches/speed_tests.rs

```diff
@@ -312,59 +325,84 @@ pub fn poplar1(c: &mut Criterion) {
     for size in test_sizes.iter() {
         group.throughput(Throughput::Bytes(*size as u64 / 8));
         group.bench_with_input(BenchmarkId::from_parameter(size), size, |b, &size| {
-            let bits = iter::repeat_with(random).take(size).collect::<Vec<bool>>();
-            let measurement = IdpfInput::from_bools(&bits);
-            let vdaf = Poplar1::new_aes128(bits.len());
-
-            b.iter(|| {
-                vdaf.shard(&measurement).unwrap();
-            });
+            let vdaf = Poplar1::new_aes128(size);
+            let mut rng = StdRng::seed_from_u64(RNG_SEED);
```

Comment:
* In the near future there will be a version of `shard()` that takes in a rand source for running test vectors, just like we have for Prio3.
* I don't understand why fixing the seed for `shard()` should improve variability. The only codepaths that *should* depend on this value in a way that impacts timing is rejection sampling, but this should be fairly unlikely. (Maybe not for 25519?)

Reply:
Yeah, I think shard should be mostly constant-time, I mainly did this for consistency of the code.